src/components/Header/index.tsx

```diff
@@ -36,16 +38,16 @@ const Header = () => {
                 : './default-avatar.png'
             }
           />
-          <Button onClick={signout} style={{ opacity: 0.9, width: 100 }}>
+          <NavButton onClick={signout} style={{ opacity: 0.9 }}>
```

Comment:
@joannerd @joesobo Why is there an inline style here? Couldn't we just put the opacity with the styled component?

Reply:
Hm, good point! For the future, should we customize a styled component with props, wrapped style components, or inline styling for one-off situations?

Reply:
I would never prefer to use inline styling for one-off situations, mostly because it's so messy.

You can use styled components to extend the styles that were previously made: https://styled-components.com/docs/basics#extending-styles

Reply:
Removed inline style completely which was overwritting NavButton opacity setting. Logout button should now be semi-transparent until hovered over just like Login button